github/repos_contents.go

```diff
@@ -91,6 +91,8 @@ func (r *RepositoryContent) GetContent() (string, error) {
 			return "", nil
 		}
 		return *r.Content, nil
+	case "none":
+		return "", fmt.Errorf("unsupported content encoding: %v, this may occur when file size > 1 MB, if that is the case consider using DownloadContents", encoding)
```

Comment:
```suggestion
		return "", errors.New("unsupported content encoding: none, this may occur when file size > 1 MB, if that is the case consider using DownloadContents")
```